app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,9 @@
+class DashboardController < ApplicationController
+
+  def index
```

Comment:
all these queries should live in their respective models rather here in the controller